reports/sdks.go

```diff
@@ -32,6 +32,15 @@ var (
 			VectorPath:   "test-vectors",
 			Type:         "web5",
 		},
+		{
+			Name:         "web5-swift",
+			Repo:         "TBD54566975/web5-swift",
+			ArtifactName: "tests",
+			FeatureRegex: regexp.MustCompile(`Web5TestVectors(\w+)`),
+			VectorRegex:  regexp.MustCompile(`(\w+)\(\)`),
+			VectorPath:   "test-vectors",
```

Comment:
I don't really understand what I'm supposed to put here. @nitro-neal or @finn-tbd would either of you know?

Reply:
You can leave this variable out. It was part of the previous system to sync vectors via pull requests, but since we're using submodules it isnt used.

Reply:
Which of these variables are you referring to? I'm not sure what any of these necessarily represent
* `FeatureRegex`
* `VectorRegex`
* `VectorPath`

Reply:
As sorry I assumed you were just asking about `VectorPath`. The other two are required. `FeatureRegex` should be a regex that will find the feature name in the first capture group when run against your JUnit-reported class names, and `VectorRegex` should be a regex that will find the vector name in your JUnit-reported test names.

Reply:
Perfecto, thanks!